label_studio/io_storages/s3/models.py

```diff
@@ -98,6 +101,11 @@ def validate_connection(self, client=None):
             logger.debug(f'Test connection to bucket {self.bucket}')
             client.head_bucket(Bucket=self.bucket)
 
+    def validate_aws_sse_kms_key_id(self):
+        if self.aws_sse_kms_key_id:
+            client = self.get_client()
+            client.describe_key(KeyId=self.aws_sse_kms_key_id)
```

Comment:
The idea here is that describe_key() will throw an exception if the key isn't valid?

Reply:
Thats right it will throw an exception if it fails, which we can catch similar to the way `storage.validate_connection()` call is throwing an http error in the `validate()` method of `S3ImportStorageSerializer`.  https://github.com/HumanSignal/label-studio/blob/a775c49585fd15bc1d99d8d44926648d9ba3951e/label_studio/io_storages/s3/serializers.py#L41

However, I just realized I'm not actually calling this method because our existing `S3ExportStorageSerializer` class is not overriding the validate method.  I'll look into adding the check in, along with a validate method.  However not sure if its worth doing, I tested adding it and validation requires we set the `region_name`, which surprisingly is not needed to use the KMS key, just validate it.

Reply:
I'm going to remove this validation method and opt not to go this route.  Reason being: an invalid KMS field already leads to an error immediately after target storage is set during the sync, so an invalid KMS field is already being caught.  The validation method wasn't actually working - as it depended on a different aws client, the kms client, which would have required I add new client creation code as well as client caching code which is pretty involved if all we're using it for is KMS key validation, which already happens anyway during sync.